src/scripts.js

```diff
@@ -110,10 +120,69 @@ function displayAllRecipes() {
   hide(favoriteRecipesSection);
   show(allRecipesSection);
 
+  removeStyling(singleRecipeView, 'single-recipe-view-alt');
+  removeStyling(allRecipesSection, 'all-recipes-view__recipes-container-alt');
+
   sortRecipesByName();
   renderRecipeCards(allRecipesContainer, recipeRepository.recipes);
 }
 
+//
+function renderWeeklyRecipeCards() {
+  const d = new Date();
+  const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+  setWeeklyRecipesDate(d, months);
+
+  setWeekGlider(generatedWeekGlider);
+  setWeekGlider(generatedWeekGlider);
+  setWeekGlider(generatedWeekGlider);
+  setWeekGlider(generatedWeekGlider);
+  setWeekGlider(generatedWeekGlider);
+  setWeekGlider(generatedWeekGlider);
+  setWeekGlider(generatedWeekGlider);
+}
+
+function randomRecipe(data) {
+  return Math.floor(Math.random() * data.length);
+}
+
+function setWeekGlider(week) {
+  let i = randomRecipe(recipesList);
+  week.innerHTML += `
+    <button class="images-container__button">
+      <p class="images-container__ingredient-name">${recipesList[i].name}</p>
+      <img class="images-container__image" src="${recipesList[i].image}" alt="image of chicken dish">
+    </button>`;
+}
+
+function addDays(aDate, numOfDays) {
+  aDate.setDate(aDate.getDate() + numOfDays);
+  return format(aDate);
+}
+
+function format(date) {
+  return [
+    ("0" + date.getDate()).slice(-2)
+  ].join('/');
+}
+
+function setWeeklyRecipesDate(d, months) {
+  generatedWeek.innerText = `Week of ${months[d.getMonth()]}, ${d.getDate()}`;
+  generatedWeek.innerText = `Week of ${months[d.getMonth()]}, ${addDays(d, 7)}`;
+  generatedWeek.innerText = `Week of ${months[d.getMonth()]}, ${addDays(d, 14)}`;
+  generatedWeek.innerText = `Week of ${months[d.getMonth()]}, ${addDays(d, 21)}`;
+}
+
+function displayWeeklyRecipes() {
+  hide(homeViewSection);
+  hide(favoriteRecipesSection);
+  hide(singleRecipeView);
+  hide(allRecipesSection);
+  show(weeklyRecipesSection);
+
+  renderWeeklyRecipeCards();
+}
```

Comment:
All of this code is unrelated to **Styling/search results** and could conflict with #48 (possibly #47). I would encourage you to save this code for later and open up a separate PR once #48 has been merged.